Applications/DataExplorer/DataView/MeshView.cpp

```diff
@@ -38,34 +38,34 @@
 #include "MeshValueEditDialog.h"
 #include "SurfaceExtractionDialog.h"
 #include "AddLayerToMeshDialog.h"
-#include "MshItem.h"
-#include "MshModel.h"
+#include "MeshItem.h"
+#include "MeshModel.h"
 
 #include "ImportFileTypes.h"
 #include "LastSavedFileDirectory.h"
 #include "SaveMeshDialog.h"
 
 
-MshView::MshView( QWidget* parent /*= 0*/ )
+MeshView::MeshView( QWidget* parent /*= 0*/ )
     : QTreeView(parent)
 {
     setUniformRowHeights(true);
     //resizeColumnsToContents();
     //resizeRowsToContents();
 }
 
-MshView::~MshView() = default;
+MeshView::~MeshView() = default;
 
-void MshView::updateView()
+void MeshView::updateView()
 {
     setAlternatingRowColors(true);
-    setColumnWidth(0,125);
+    setColumnWidth(0,150);
```

Comment:
Is this size change intended?

Reply:
:-/

It's from a different branch. Not sure how it slipped in there.  
It doesn't matter, though...